test/C/src/TriggerIssue.lf

```diff
@@ -0,0 +1,38 @@
+/** Destination cannot react to y. Only if x is also a trigger (uncomment reaction) it will work. */
```

Comment:
This comment seem wrong? What is the test meant to capture? The output `x` is never set, but it's also not connected. An output is produced on `y`, and the downstream reactor has a reaction that is triggered by it. Did this not work prior to the change?

Reply:
Good catch!  This comment was left over from the issue.  Fixed in d6d6cd6deee29db19ec08812f37cd67628bb5c6f.